src/EventStore.Core/Services/Transport/Tcp/ClientWriteTcpDispatcher.cs

```diff
@@ -0,0 +1,256 @@
+﻿using System;
```

Comment:
Would you mind explaining why we need this?

Reply:
Please see the commit message: adb35eb310d7c8c5c78440fa01008d463e2abd6b
Client operations should not be allowed on the internal TCP interface.